_includes/tutorials/message-ordering/kafka/code/src/test/java/io/confluent/developer/KafkaProducerApplicationTest.java

```diff
@@ -0,0 +1,47 @@
+package io.confluent.developer;
```

Comment:
Is this intended to show up somewhere in the tutorial? I didn't see it on staging.

Reply:
Hmmm, not sure about this one.

Reply:
The project this KT was cloned from has a different test section that uses a unit test.   Part of the build runs only the unit tests, so it might be a good idea to leave it in.